packages/interfaces/src/protocols.ts

```diff
@@ -101,7 +101,7 @@ export type Callback<T extends IDecodedMessage> = (
   msg: T
 ) => void | Promise<void>;
 
-export enum SendError {
+export enum ProtocolError {
```

Comment:
This one I'd like to discuss. 

This `enum` is used as a `constant` and part of runtime and not `typescript` compile time. 
That means that in some cases `@waku/interfaces` should be `dependencies` and not `devDependencies`. 

So far we didn't see any issues because of that and I was not able to get any in my experiments, but overall it seems to me worth making a part of `@waku/core`. 

wdyt @waku-org/js-waku-developers ?

Reply:
@weboko can we track this in an issue? We can discuss this in the coming PM call, or async within the issue
cc @adklempner

Reply:
sure, I wanted to bring this to discussion so that our team is aware of it